zerver/worker/queue_processors.py

```diff
@@ -491,7 +491,11 @@ def consume(self, event):
         # TODO: Do we actually want to allow multiple Services per bot user?
         services = get_bot_services(user_profile_id)
         for service in services:
-            get_bot_handler(str(service.name)).handle_message(
-                message=message,
-                bot_handler=self.get_bot_api_client(user_profile),
-                state_handler=self.get_state_handler())
+            bot_handler = get_bot_handler(str(service.name))
+            if bot_handler is None:
+                logging.warning("Sorry! Service not available. Please select a service from the dropdown list of services only.")
```

Comment:
this error message is for the sysadmin, not the person who chose the service, so I think we want text more like this:

`logging.error("Error: User %s has embedded bot service with invalid bot %s" % (user_profile_id, service.name))`